dds_web/api/dds_decorators.py

```diff
@@ -84,6 +85,8 @@ def make_commit(*args, **kwargs):
         # Run function, catch errors
         try:
             result = func(*args, **kwargs)
+        except (pymysql.err.OperationalError, sqlalchemy.exc.SQLAlchemyError) as err:
```

Comment:
As I learned while writing the code for the file removal, it's actually `sqlalchemy.exc.OperationalError` that is thrown, even though `pymysql.err.OperationalError` is the one shown.

Reply:
Oh ok! Thanks, I was wondering about this. I'll change it.

Reply:
```suggestion
        except sqlalchemy.exc.OperationalError as err:
```